src/main/kotlin/com/hibob/academy/employeeFeedback/Service/SessionJwtService.kt

```diff
@@ -18,6 +19,8 @@ class SessionJwtService(private val employeeDao: EmployeeDao) {
     private val secretKey = "1212121221212121212121212213213233213131232321"
 
     fun createJwtToken(userLoggedInDetails: UserLoggedInDetails): String {
```

Comment:
I'm not sure about the flow, but if you already got userLoggedInDetails, doesn't it mean the user exists?